src/schneider_lab_to_nwb/schneider_2024/schneider_2024_metadata.yaml

```diff
@@ -52,4 +52,24 @@ Ecephys:
     # - name: ch
     #   description: The channel label of the best channel, as defined by the user.
     # - name: sh
-    #   description: The shank label of the best channel.
\ No newline at end of file
+    #   description: The shank label of the best channel.
+Behavior:
+  TimeSeries:
+    - name: encoder
+      description: Sampled values for entire duration of experiment for lever pressing/treadmill behavior read from a quadrature encoder.
+    - name: lick
+      description: Samples values for entire duration of experiment for voltage signal readout from an infrared/capacitive) lickometer sensor.
```

Comment:
Maybe it would make sense to add devices for each of them if you have info.

Reply:
Good idea! I will add this to the list of data/metadata requests from the Schneider lab.